schemas/job_locale.yaml

```diff
@@ -28,9 +28,10 @@ job_locale:
       name: My Job 1
       state: completed
     users:
-    - id: abcd1234cdef1234abcd1234cdef1234
+      id: abcd1234cdef1234abcd1234cdef1234
```

Comment:
I think the dash is required as it's a list right?

Reply:
good point 👍  thanks, returned back